javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -7,6 +7,44 @@ import { useAuth0 } from "@auth0/auth0-react";
 import { fetchWithToken } from "main/utils/fetch";
 import NavbarHover from "../Nav/NavbarHover";
 
+export function PureNavbar({ isAdmin, isMember, ChannelPages, AdminPages, SearchPages, DataPages }) {
+  return (
+    <Navbar bg="dark" variant="dark" expand="lg">
+      <Navbar.Toggle />
+      <Navbar.Collapse>
+        <LinkContainer to={""}>
```

Comment:
While you're refactoring these components, could you also clean up cases of `prop={"literal"}`, which can be replaced with `prop="literal"`?

Reply:
Done!